### DISCUSSION TITLE
Apple Silicon support?

### DISCUSSION THREAD

**Discussion by nifte** [Kong/insomnia#4393] [Help] [answered]
It seems like Insomnia already runs on Electron 11+, is there any way we could get an Apple Silicon build?

**Comment by dimitropoulos** [comment, marked as answer]
should be able to deliver on this soon.  please check out https://github.com/Kong/insomnia/pull/4140 and https://github.com/Kong/insomnia/issues/2964#issuecomment-1004967388
